system/Commands/Translation/LocalizationFinder.php

```diff
@@ -172,14 +186,16 @@ private function findTranslationsInFile($file): array
         preg_match_all('/lang\(\'([._a-z0-9\-]+)\'\)/ui', $fileContent, $matches);
```

Comment:
There is one feature: The regular expression initially discards bad keys. They will not be shown in the list.